src/test/scala/play/api/cache/redis/connector/RedisMasterSlavesSpec.scala

```diff
@@ -0,0 +1,118 @@
+package play.api.cache.redis.connector
+
+import com.dimafeng.testcontainers.{Container, DockerComposeContainer, ExposedService}
+import org.apache.pekko.actor.ActorSystem
+import org.scalatest.BeforeAndAfterAll
+import org.testcontainers.containers.wait.strategy.Wait
+import play.api.cache.redis.configuration.{RedisHost, RedisMasterSlaves, RedisSettings}
+import play.api.cache.redis.impl.{LazyInvocation, RedisRuntime}
+import play.api.cache.redis.test.{Helpers, IntegrationSpec, StoppableApplication}
+import play.api.cache.redis.{LogAndFailPolicy, RedisConnector}
+
+import java.io.File
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.concurrent.{ExecutionContext, Future}
+
+class RedisMasterSlavesSpec extends IntegrationSpec with BeforeAndAfterAll {
+
+  private val host = "localhost"
+  private val masterPort = 6379
+  private val slavePort = 6479
+
+  private val containerDef = DockerComposeContainer.Def(
+    new File("src/test/resources/docker-compose.yml"),
+    tailChildContainers = true,
+    exposedServices = Seq(
+      ExposedService("redis-master", masterPort, Wait.forLogMessage(".*Ready to accept connections tcp.*\\n", 1)),
+      ExposedService("redis-slave", slavePort, Wait.forLogMessage(".*MASTER <-> REPLICA sync: Finished with success.*\\n", 1)),
+    ),
+  )
+
+  private def startContainer = containerDef.start()
+
+  private lazy val container: Container = startContainer
+
+  override protected def beforeAll(): Unit = container
+
+  override protected def afterAll(): Unit = container.stop()
```

Comment:
Could you refactor it to `MasterSlaveContainer` trait in `test` package for greater consistency? You can get inspiration from

https://github.com/KarelCemus/play-redis/blob/master/src/test/scala/play/api/cache/redis/test/RedisClusterContainer.scala

or 

https://github.com/KarelCemus/play-redis/blob/master/src/test/scala/play/api/cache/redis/test/RedisStandaloneContainer.scala

or 

https://github.com/KarelCemus/play-redis/blob/master/src/test/scala/play/api/cache/redis/test/RedisSentinelContainer.scala

Reply:
Hello, @KarelCemus!  Yes, I'll refactor it.

Reply:
Hello, @KarelCemus! I have refactored the tests.